### PR TITLE
Fix to Issues caused by TopNotch on Android

### PR DESCRIPTION
**Pull request by NicolasChampagne**
## PR Type

What kind of change does this PR introduce?
- Bugfix

## What is the current behavior?

Translucent and nav and status bar that overlaps page
Scrolling issues due to requestLayout() in an infinite loop
Keyboard Flys out on nav bar change

## What is the new behavior?

Proper status and nav bar behaviour
No scrolling issues
No Keyboard flyout

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] Tested code with current [supported SDKs](../README.md#supported)Test Run` results.
- [x] Contains **NO** breaking changes
- [x] Associated with an issue (GitHub or internal)

Internal Issue (If applicable):
https://nventive.visualstudio.com/Umbrella/_workitems/edit/158376
https://nventive.visualstudio.com/Umbrella/_workitems/edit/158363
https://nventive.visualstudio.com/Umbrella/_workitems/edit/158377
https://nventive.visualstudio.com/Umbrella/_workitems/edit/158567

### CODE REVIEW

**Comment by MatFillion** [comment]
Looks too simple to be true 👍

File: src/Uno.UI.BindingHelper.Android/Uno/UI/UnoViewGroup.java [hunk 1]
```diff
@@ -147,30 +146,11 @@ public final void nativeFinishLayoutOverride(){
...
-			try
-			{
-				_isInOnLayoutCore = true;
-
-				_childrenNeedingForceLayout.clear();
-				onLayoutCore(changed, left, top, right, bottom);
-
-				if (!_childrenNeedingForceLayout.isEmpty()) {
-					for (int i = 0; i < _childrenNeedingForceLayout.size(); i++) {
-						_childrenNeedingForceLayout.get(i).forceLayout();
```
Comment (davidjohnoliver) [inline src/Uno.UI.BindingHelper.Android/Uno/UI/UnoViewGroup.java]: @YGuerin - do you recall why this code was added? Was it something about VisibleBoundsPadding not updating properly in some circumstance? 

@NicolasChampagne - can you explain why you're removing it?

Comment (NicolasChampagne) [reply src/Uno.UI.BindingHelper.Android/Uno/UI/UnoViewGroup.java]: After testing on devices without this code VisibleBoundsPadding was being updated correctly.

It was also causing an infinite loop for some listviews, making apps lag

**Pull request merged**